<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-ANBA6IRE" name="Default test" tests="3" failures="1" timestamp="5 Jul 2020 12:16:01 GMT" time="6.154" errors="0">
  <testcase name="@AfterMethod closeIrct" time="0.0" classname="testcases.TC001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at wrappers.GenewirWrappers.closeAllBrowsers(GenewirWrappers.java:718)
	at wrappers.ProjectWrappers.closeIrct(ProjectWrappers.java:13)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:703)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
    </failure>
  </testcase> <!-- @AfterMethod closeIrct -->
  <testcase name="@BeforeMethod launchIrctc" time="0.0" classname="testcases.TC001">
    <skipped/>
  </testcase> <!-- @BeforeMethod launchIrctc -->
  <testcase name="@AfterMethod closeIrct" time="0.0" classname="testcases.TC001">
    <skipped/>
  </testcase> <!-- @AfterMethod closeIrct -->
  <testcase name="@BeforeMethod launchIrctc" time="0.0" classname="testcases.TC001">
    <skipped/>
  </testcase> <!-- @BeforeMethod launchIrctc -->
  <testcase name="@AfterMethod closeIrct" time="0.0" classname="testcases.TC001">
    <skipped/>
  </testcase> <!-- @AfterMethod closeIrct -->
  <testcase name="irctcRegistration" time="0.005" classname="testcases.TC001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at wrappers.GenewirWrappers.clickByXpath(GenewirWrappers.java:281)
	at testcases.TC001.irctcRegistration(TC001.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
    </failure>
  </testcase> <!-- irctcRegistration -->
  <testcase name="irctcRegistration" time="0.001" classname="testcases.TC001">
    <skipped/>
  </testcase> <!-- irctcRegistration -->
  <testcase name="irctcRegistration" time="0.0" classname="testcases.TC001">
    <skipped/>
  </testcase> <!-- irctcRegistration -->
</testsuite> <!-- Default test -->
